### DISCUSSION TITLE
Bayesian Hierarchical Modeling in Orbit

### DISCUSSION THREAD

**Discussion by brnjemison** [uber/orbit#754] [Q&A]
Hello,

I'm new to Bayesian stats and the Orbit library.  Are there ways of creating hierarchical models in Orbit? For example, if I have a dataset similar to the iclaims dataset used in the Quick Start guide, however, an additional column for classification (i.e. state). I would like to use the knowledge from states with more extensive time series data to help the inform the posterior for states with less data. I've seen a handful of examples of this performed in PyMC3 and NumPyro and was wondering if there is similar functionality in Orbit. If so is there any documentation or tutorial where I can read up on this?
Thank you in advance.

**Comment by rjscout24** [comment]
you can create additional variables for your states, and then use the global estimates / guess as the prior along with appropriate distribution, as it has been shown also in pymc3 docs.

**Comment by edwinnglabs** [comment]
@brnjemison  are you doing the hierarchy on regression or the trend (or both?)
